doc/cephfs/mount-using-fuse.rst

```diff
@@ -2,57 +2,24 @@
  Mount CephFS using FUSE
 ========================
 
-Prerequisite
-------------
-Before mounting CephFS, ensure that the client host (where CephFS has to be
-mounted and used) has a copy of the Ceph configuration file (i.e.
-``ceph.conf``) and a keyring of the CephX user that has CAPS for the Ceph MDS.
-Both of these files must be present on the host where the Ceph MON resides.
+Pre-requisites
+==============
```

Comment:
Add a link to the new `mount.rst` doc in case someone comes to this page directly from Google.

Reply:
Added the link.